pkgs/by-name/ws/wslay/package.nix

```diff
@@ -12,22 +20,21 @@ stdenv.mkDerivation rec {
   };
 
   strictDeps = true;
+  postPatch = ''
+    sed -i 's!add_stylesheet!add_css_file!' doc/sphinx/conf.py.in
+  '';
 
-  nativeBuildInputs = [ cmake ];
-
-  cmakeFlags = [
-    (lib.cmakeBool "WSLAY_TESTS" true)
-  ];
+  nativeBuildInputs = [autoreconfHook pkg-config sphinx];
```

Comment:
This doesn't comply with [RFC 0166](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md#attribute-sets-and-lists). Are you using Alejandra? I'd suggest using `nixfmt-rfc-style` for PRs to Nixpkgs.

Reply:
I didn't realize that. Now the code is formatted by nixfmt-rfc-style.